### ISSUE TITLE
Taking off SG armor/smartgun does not remove the nightvision goggles

### ISSUE THREAD

**Issue by Alekshhh** [RMC-14/RMC-14#4761]
they stay on and give the full effect, despite the gun needing to be on to wear them

**Comment by Dutch-VanDerLinde** [comment]
In CM the goggles stay on but they turn off if you're unskilled

**Comment by Skarletto** [comment]
> In CM the goggles stay on but they turn off if you're unskilled

this is correct though I'm not sure if it's intended in cm13 either, though whats intended or not in cm13 is not my business teehee

**Comment by Dutch-VanDerLinde** [comment]
This is fixed latest update, SGO night vision now auto turns off if you dont have the right skills
